model/search_pipeline/get/structures.smithy

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetSearchPipeline_Input {
+    @required
+    @httpLabel
+    pipeline: String,
+}
+
+structure GetSearchPipeline_Output {
+    @httpPayload
+    content: SearchPipelineStructure
```

Comment:
`GET /_search/pipeline` and `GET /_search/pipeline/{pipeline}` both return a _MAP_ of search pipelines.